### PR TITLE
feat(HMS-1772): Allow selecting custom resource group

### PR DESCRIPTION
**Pull request by ezr-ondrej**
[Screencast from 2023-10-05 20-27-33.webm](https://github.com/RHEnVision/provisioning-frontend/assets/2884324/7531a879-93a0-48c4-aec0-9427479d23f8)

Simple and straightforward I hope :)

UI part for https://github.com/RHEnVision/provisioning-backend/pull/694
Update your backend env or test with stage! :)

### CODE REVIEW

File: src/mocks/utils.js [hunk 1]
```diff
@@ -5,14 +5,19 @@ import { render } from '@testing-library/react';
...
   const queryClient = new QueryClient({
     logger: {
       log: console.log,
       warn: console.warn,
       // ✅ no more errors on the console
       error: () => {},
     },
+    defaultOptions: {
+      queries: {
+        retry: queryRetries,
```
Comment (ezr-ondrej) [inline src/mocks/utils.js:18]: This is good practice in tests to have easier time to test for failures.

**Comment by lzap** [comment]
Nice.

**Comment by akhil-jha** [comment]
Ah, this would require wizard changes in QE tests since we have a new dropdown.

**Comment by akhil-jha** [comment]
@ezr-ondrej Can please you rebase this? Thanks!

**Comment by ezr-ondrej** [comment]
> @ezr-ondrej Can please you rebase this? Thanks!

rebased! :) I think that only now we will run the Azure tests, right?

**Review by akhil-jha — APPROVED** [review]

**Comment by ezr-ondrej** [comment]
I'll go ahead and self merge, I promise to fix any issues that may arrise, but at least now we've tested this with Azure tests on :)

**Pull request merged**